src/qt/clientmodel.h

```diff
@@ -88,7 +88,8 @@ class ClientModel : public QObject
     PeerTableModel *peerTableModel;
     BanTableModel *banTableModel;
 
-    QTimer *pollTimer;
+    QTimer *pollTimer1;
+    QTimer *pollTimer2;
```

Comment:
For readability, I think it would make more sense to name these timers `pollTimer_TPS` and `pollTimer_Ban`, since the original deals with updating the transactions per second and the new one deals with clearing expired bans.